src/web/app/pages-instructor/instructor-search-page/student-result-table/student-result-table.component.html

```diff
@@ -1,5 +1,5 @@
 <div *ngFor="let course of studentTables" class="card border border-light-blue mb-3 student-course-table">
-  <div class="card-header alert-primary font-weight-bold text-break">[{{course.courseId}}]</div>
+  <div class="card-header alert-primary font-weight-bold text-break" [innerHtml]="course.courseId | highlighter:searchedTerm:true"></div>
```

Comment:
Seems that the square brackets were accidentally removed here. Let's not introduce unintended changes, can you add the brackets back?